scripts/post-update.sh

```diff
@@ -139,25 +139,20 @@ if ! ssh-keygen -H -F github.com; then
     fi
 fi
 
-cd /home/pi/companion
-echo 'Checking ping-python status...'
-PING_STATUS=$(git submodule status | grep ping-python | head -c 1)
-if [[ ! -z $PING_STATUS && ($PING_STATUS == '+' || $PING_STATUS == '-') ]]; then
-    echo 'ping-python needs update.'
-    git submodule update --recursive -f submodules/ping-python
-    echo 'Installing ping-python...'
-    cd /home/pi/companion/submodules/ping-python
-    sudo python setup.py build install
-    if [ $? -ne 0 ] # If "pip install ping-python" failed:
+# install bluerobotics-ping if neccessary
+if pip list | grep bluerobotics-ping; then
+    echo 'bluerobotics-ping already installed'
+else
+    echo 'installing bluerobotics-ping...'
+    sudo pip install bluerobotics-ping
+    if [ $? -ne 0 ] # If "pip install pynmea2" failed:
     then
```

Comment:
You can do `]; then` with the comment in the previous line to make it more readable.